core/src/main/resources/defaults/blueprints/cdp-sdx-ha.bp

```diff
@@ -374,31 +308,43 @@
       },
       {
         "refName": "zookeeper",
-        "serviceType": "ZOOKEEPER",
         "roleConfigGroups": [
           {
+            "base": true,
             "refName": "zookeeper-SERVER-BASE",
-            "roleType": "SERVER",
-            "base": true
+            "roleType": "SERVER"
           }
-        ]
+        ],
+        "serviceType": "ZOOKEEPER"
       },
       {
-        "serviceType": "KNOX",
         "refName": "knox",
         "roleConfigGroups": [
           {
             "base": true,
+            "configs": [
+              {
+                "name": "gateway_master_secret",
```

Comment:
Please don't add these hard-coded values in `master`.  There is no need for them in recent CB versions.  If you would like to use the blueprint in environment running older CB, just add the secret in the file you use to create the blueprint there.

Reply:
Does CB in mow-int inject this configuration automatically now?

Reply:
Yes.